src/ddl/covid_hosp.sql

```diff
@@ -500,15 +489,151 @@ CREATE TABLE `covid_hosp_state_timeseries` (
   `total_patients_hospitalized_confirmed_influenza_coverage` INT,
   PRIMARY KEY (`id`),
   -- for uniqueness
-  -- for fast lookup of most recent issue for a given state, date, and record type
-  UNIQUE KEY `issue_by_state_and_date` (`state`, `date`, `issue`, `record_type`),
-  -- for fast lookup of a time-series for a given state, issue, and record type
-  KEY `date_by_issue_and_state` (`issue`, `state`, `date`, `record_type`),
-  -- for fast lookup of all states for a given date, issue, and record_type
-  KEY `state_by_issue_and_date` (`issue`, `date`, `state`, `record_type`)
+  -- for fast lookup of most recent issue for a given state and date
+  UNIQUE KEY `issue_by_state_and_date` (`state`, `date`, `issue`),
+  -- for fast lookup of a time-series for a given state and issue
+  KEY `date_by_issue_and_state` (`issue`, `state`, `date`),
+  -- for fast lookup of all states for a given date and issue
+  KEY `state_by_issue_and_date` (`issue`, `date`, `state`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
 
 
+/*
+`covid_hosp_state_timeseries` stores the versioned "state timeseries" dataset,
+which contains data from the daily snapshot files.
+Schema is equivalent to `covid_hosp_state_timeseries`.
+*/
+CREATE TABLE `covid_hosp_state_daily` (
```

Comment:
since this essentially duplicates `covid_hosp_state_timeseries`, you can use a shortcut to create it...  see [v4_schema.sql](https://github.com/cmu-delphi/delphi-epidata/blob/4e46532388de61e30ed010334c4ad0f0f11f4749/src/ddl/v4_schema.sql#L54-L59) for an example.

Reply:
Tried this approach out, it works fine in the sandbox but causes some issues when trying to re-run `acquisition` on the tables - https://github.com/cmu-delphi/delphi-epidata/actions/runs/4681152223/jobs/8293372868?pr=1126#step:16:84. I think it's fine to stick with two table definitions even if repetitive

Reply:
thats concerning.  i think you just forgot a `ALTER TABLE covid_hosp_state_daily MODIFY id INT NOT NULL AUTO_INCREMENT;`

while it is fine to have separate and full table definitions in this file (in fact, it eventually will be desired (if not required!) when this file becomes auto-generated), we should endeavor to use the shorthand in the migration file.  there it serves as insurance against transcription/copypasta errors, while making the intent clearer and the entirety of the process more understandable.